src/export/export-css.ts

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs-extra';
+import StyleDictionary from 'style-dictionary';
+
+import { convertShadowsDesignTokenToCss } from '../utils/convertDesignTokenToCss';
+import type { Tokens } from '../utils/types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const reshapeDesignTokens = (obj: any) => {
```

Comment:
can you tell me a little more about what reshapeDesignTokens does in the exporters?

Reply:
Essentially it's changing the structure (schema?) of the design tokens (deleting unneeded or unsupported tokens, changing key names, etc.) for when they're being passed into Style Dictionary to be turn into the final output, in this case CSS custom properties.

Reply:
ok I thought maybe we had the "generic/standard" token format we could pass to Style Dictionary directly? We still need to do a bit of manipulation for each type?

Reply:
Here's a link to an audio reply: https://portablestudios.slack.com/files/U02F96MDWJH/F04A3FKEX42/audio_message.webm

Reply:
An audio reply? 😂 I can't open it 😢

Reply:
https://user-images.githubusercontent.com/63755046/205546485-ba684bb6-c289-47c9-9b14-d54debfc29b7.mp4

How about now?